Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -49,6 +49,9 @@
 "account-creation-captcha-required" = "CAPTCHA verification required.";
 "account-creation-captcha-retry" = "Incorrect entry. Please try again.";
 "account-creation-captcha-obtaining" = "Obtaining a new CAPTCHA...";
+"account-creation-captcha-title" = "CAPTCHA security check";
+"account-creation-captcha-cannot-see-image" = "Can't see the image? $1";
+"account-creation-captcha-request-account" = "Request an account.";
 "account-creation-logging-in" = "Logging in...";
```

Comment:
Quick note for Andy: localized strings added to this file get automatically added to translatewiki.net. We periodically get translations back - about weekly, as PRs, which are then tested (occasionally translators will accidentally add things like html comments which are fine if the string was going to used with html for a web page but which don't make sense for the native app).

Here's a link to the iOS TWN page: https://translatewiki.net/w/i.php?title=Special:Translate&group=out-wikimedia-mobile-wikipedia-ios&filter=translated&language=de&action=translate
Usually we don't have to do anything with this, but it's nice to see where the translations actually happen.